libs/geoservices/covid/src/lib/pages/maps/maps.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'tamu-gisc-maps',
+  templateUrl: './maps.component.html',
+  styleUrls: ['./maps.component.scss']
+})
+export class MapsComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
```

Comment:
All class members should have member access defined `public`, `private`, `protected`.

If you hit F1 in VSCode, and type out `Fix All`, VSCode will automatically append the most appropriate access type based on class usage. Defining these will set constraints when extending classes or when certain properties should not be used in templates.